src/app.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+require('dotenv').config();
+const Database = require('./db');
+
+const app = express();
+const devPort = process.env.PORT;
+const prodPort = process.env.PRODUCTION_PORT;
+
+app.use(express.json());
+
+if (process.env.NODE_ENV === 'development') {
+  const db = new Database(process.env.DB_DEV_URL);
+  db.connect();
+
+  app.listen(devPort, () => {
+    console.log(`Server is listening on port: ${devPort}`);
+  });
+}
+
+if (process.env.NODE_ENV === 'production') {
+  const db = new Database(process.env.DB_PRODUCTION_URL);
+  db.connect();
+
+  app.listen(prodPort, () => {
+    console.log(`Server is listening on port: ${prodPort}`);
+  });
+}
```

Comment:
This code can be made a bit more DRY.

You can set the server port as:
```js
const port = process.env.NODE_ENV === 'development' ? process.env.DEVELOPMENT_PORT : process.env.PRODUCTION_PORT;
```

Similarly set the DB URl and then you can write just one code block for starting the server and connecting the DB.